EcdarProtoBuf/services.proto

```diff
@@ -14,4 +14,5 @@ import "google/protobuf/empty.proto";
 
 service EcdarBackend {
     rpc UpdateComponents(ComponentsUpdateRequest) returns (google.protobuf.Empty);
+    rpc SendQuery(Query) returns (QueryResponse);
```

Comment:
This is a bit of a hacky solution, as some queries can take a very long time to complete so it depends on the client not having any response timeouts.

Another solution could be having the server return a stream of QueryUpdates and finally a QueryResult when done.
A QueryUpdate would contain information like how many states have been walked through similar to what UPPAAL shows and act to reset the response timeout similar to a heartbeat message.
The QueryResult would contain the result of the query.